Finished index.html and about.html

jacob-h97 opened the pull request:
Corrected missing Head Nav bar and styled about.html as requested.

jacob-h97 committed:
Finished index.html and about.html

Corrected missing Head Nav bar and styled about.html as requested.

fivegoblins reviewed (changes requested):
**MVP**:

**Home page**:

- You did a great job of matching the design file using HTML and CSS. You followed the instructions to complete the box challenge and gave all of the boxes the appropriate color.

**About page**:

- You did a great job here also of matching the design file as closely as possible. Nice work.

**Self Study Questions**:

- I only see 3 modified files on this pull request; the index.html and about.html files and the index.css. I'm not seeing anywhere that you answered the questions. This is required to complete the sprint challenge.

**Rating: 1** (make changes and resubmit)

**Note**: You should be making frequent commits (every time you finish a task), and your commit messages should be descriptive.

jacob-h97 committed:
Added Questions.txt

This update adds the answer's to the Self Study questions the file is in question.txt
